index.js

```diff
@@ -28,10 +28,17 @@ function getModules() {
   eslintMod.filename = eslintLoc;
   eslintMod.paths = Module._nodeModulePaths(path.dirname(eslintLoc));
 
-  var Definition = eslintMod.require("escope/lib/definition").Definition;
-  var escope  = eslintMod.require("escope");
+  try {
```

Comment:
Not a blocker, but my preference would be to not rely on `var` hoisting here, but it looks like that's the style that was used here previously. babel-eslint actually supports Node >=4, so we can update some of this at some point!